pkg/cmd/extension/command.go

```diff
@@ -165,7 +167,7 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 			return cmd
 		}(),
 		&cobra.Command{
-			Use:   "remove <name>",
+			Use:   "remove <name> [--precompiled]",
```

Comment:
Does `remove` support `--precompiled` now? How does it make use of it?

Reply:
that's just an error D: